packages/salesforcedx-vscode-core/package.json

```diff
@@ -23,6 +23,7 @@
   },
   "categories": ["Other"],
   "dependencies": {
+    "@salesforce/core": "^0.18.2",
```

Comment:
@DatGuyJonathan, @lcampos - This is an important change since it fulfills one of my architectural goals on making the VS Code commands faster – use the in-memory version of @salesforce/core instead of delegating to the CLI whenever possible. This can potentially shave 1.5 seconds from invocations.

What Alli is doing can be the stepping stone toward using this approach later.

Reply:
@vazexqi Just curious, where does the 1.5 second statistic come from?